devel/darwinbuild-legacy/Portfile

```diff
@@ -1,53 +1,51 @@
 PortSystem      1.0
+PortGroup       github 1.0
 
 name            darwinbuild-legacy
 version         0.8.0
-revision	655
 categories      devel
 platforms       darwin
+license         BSD
 maintainers     wms
 description     Darwinbuild is a set of tools that facilitate building the sources \
-		released by Apple.
+                released by Apple.
 
 long_description \
-		The Darwin Build Scripts are a collection of tools that assist compilation of \
-		the many projects contained in Darwin, the open source base of Apple's \
-		Mac OS X operating system.  Apple publishes the sources of these projects \
-		in an archive format (.tar.gz).  An archive is published for each project \
-		version on Apple's site. These tools will provide the proper build environment as well as help to \
-		resolve any necessary dependencies prior to building.
-
-homepage        http://darwinbuild.macosforge.org/
-master_sites    http://svn.macosforge.org/repository/darwinbuild/
+        The Darwin Build Scripts are a collection of tools that assist compilation of \
+        the many projects contained in Darwin, the open source base of Apple's \
+        Mac OS X operating system.  Apple publishes the sources of these projects \
+        in an archive format (.tar.gz).  An archive is published for each project \
+        version on Apple's site. These tools will provide the proper build environment as well as help to \
+        resolve any necessary dependencies prior to building.
 
 pre-fetch {
     if {${os.major} > 9} {
         return -code error "The ${name} port is only for Darwin 8 and 9. For Darwin\
-			    10 and later, install the darwinbuild port." 
+                10 and later, install the darwinbuild port."
     }
 }
 
+
 platform darwin 9 {
-	worksrcdir	Darwin9
-	svn.url         ${master_sites}/releases/${worksrcdir}
+    github.setup    macosforge darwinbuild 4848b7f5b223e094a26c413b451d27b938b0f2bd
```

Comment:
Doesn't this override the version? You should probably set the version after `github.setup`.